generate.go

```diff
@@ -2,41 +2,31 @@ package swagno
 
 import (
 	"encoding/json"
-	"fmt"
 	"log"
-	"reflect"
-	"strconv"
 	"strings"
 
-	"github.com/go-swagno/swagno/components/definition"
 	"github.com/go-swagno/swagno/components/endpoint"
 	"github.com/go-swagno/swagno/components/mime"
 	"github.com/go-swagno/swagno/components/parameter"
+	"github.com/go-swagno/swagno/generator"
 	"github.com/go-swagno/swagno/http/response"
 )
 
-func hasStructFields(s interface{}) bool {
-	rv := reflect.ValueOf(s)
-
-	if rv.Kind() != reflect.Struct {
-		return false
-	}
-
-	numFields := rv.NumField()
-	return numFields > 0
-}
-
 func appendResponses(sourceResponses map[string]endpoint.JsonResponse, additionalResponses []response.Info) map[string]endpoint.JsonResponse {
-	for _, response := range additionalResponses {
+	responseGenerator := generator.NewResponseGenerator()
```

Comment:
to keep it idiomatic to go object creation and make less redundant can we call this as `generator.New()`?